esrally/utils/git.py

```diff
@@ -62,31 +62,30 @@ def fetch(src, remote="origin"):
 
 
 @probed
-def checkout(src_dir, branch="master"):
+def checkout(src_dir, *, branch):
     if process.run_subprocess_with_logging("git -C {0} checkout {1}".format(io.escape_path(src_dir), branch)):
         raise exceptions.SupplyError("Could not checkout [%s]. Do you have uncommitted changes?" % branch)
 
 
 @probed
-def rebase(src_dir, remote="origin", branch="master"):
-    checkout(src_dir, branch)
+def rebase(src_dir, remote="origin", *, branch):
```

Comment:
Should we force the use of kwargs for `remote` as well (i.e. move `*` before `remote`?)

Reply:
I considered it but did not want to clutter the code unnecessarily or make this pull request bigger. Now that I reconsider it again I guess it would be closer to the way we use the git cli?

I don't have a strong opinion here, I can do it if you prefer

Reply:
> Now that I reconsider it again I guess it would be closer to the way we use the git cli?

Yeah that's how I am thinking as well.

Reply:
We discussed offline, @pquentin will tackle making `remote` a mandatory kw arg without a default value (as done in `branch` here) in a follow up PR.